source/erupted/types.d

```diff
@@ -4038,14 +4038,13 @@ version( VK_USE_PLATFORM_XCB_KHR ) {
 
 // VK_KHR_wayland_surface
 version( VK_USE_PLATFORM_WAYLAND_KHR ) {
-	public import wayland.client;
+	public import wayland.native.client;
 
 	enum VK_KHR_WAYLAND_SURFACE_SPEC_VERSION = 6;
 	enum VK_KHR_WAYLAND_SURFACE_EXTENSION_NAME = "VK_KHR_wayland_surface";
 
 	alias VkWaylandSurfaceCreateFlagsKHR = VkFlags;
-	struct wl_display;
-	struct wl_surface;
+	alias wl_surface = wl_proxy;
```

Comment:
VkWaylandSurfaceCreateInfoKHR has a field of type wl_surface* but you have removed the definition of that type, how can this work?

Reply:
`wl_display` is imported by `wayland.native.client`.
Having a local definition makes erupted expecting `erupted.types.wl_display`, whereas application code is supplying `wayland.native.client.wl_display`

Reply:
Sorry for the late reply. I have no opportunity to test this, but I assume that it works at your side, hence I will trust you.
Btw, I don't like the fact that erupted has so many dependencies even if not used. I am working on (or rather postponing ...) a version which does have no dependencies at all, and all the platform specific code is implemented as several template mixins. So its in the users hand to decide which libraries he wants to use. What do you think about this approach?